server/utils/utils.ts

```diff
@@ -206,6 +230,13 @@ const isPassedArdOrCrd = (licence: LicenceSummary, prisoner: Prisoner | PrisonAp
   return false
 }
 
+const releaseDateLabel = (licence: Licence, nomisRecord: Prisoner): string => {
+  if (licence) {
+    return licence.actualReleaseDate ? 'Confirmed release date' : 'CRD'
+  }
+  return nomisRecord.confirmedReleaseDate ? 'Confirmed release date' : 'CRD'
```

Comment:
is this right? the labels are the same?

Reply:
Yes, `actualReleasedate` on the licence and `confirmedReleaseDate` in NOMIS are the same thing.
The purpose of this ticket is to read the licence dates if they exist, and if not to fall back to NOMIS. Otherwise we end up in a situation where the dates on the licence can end up being different to the dates we show in CVL, eg if the dates are deleted in NOMIS and we retain them for IS91 cases (as the licence generation reads from the CVL database)